Description: add scripts for task
 tree (1.8.0-2ubuntu1) jammy; urgency=medium
   * add script for task
Author: Lok Wing POON <poonlokwing@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2024-03-29

--- /dev/null
+++ tree-1.8.0/script/testing.sh
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "this is a test from Lok Wing Poon"
+
